webnn/resources/utils.js

```diff
@@ -282,6 +282,30 @@ const getReductionPrecisionTolerance = (resources, operationName) => {
   return tolerance;
 };
 
+/**
+ * Get ULP tolerance of resample2d operations.
+ * @param {Object} resources - Resources used for building a graph
+ * @param {String} operationName - An operation name
+ * @returns {Number} A tolerance number
```

Comment:
```suggestion
 * @returns {Number} A tolerance number in ULP
```
(to clarify it's not absolute difference, because 84 would be a huuuge tolerance 😉)